tap_stripe/__init__.py

```diff
@@ -428,6 +437,16 @@ def get_bookmark_for_stream(stream_name, replication_key):
             int(utils.strptime_to_utc(Context.config["start_date"]).timestamp())
     return stream_bookmark
 
+def get_bookmark_for_sub_stream(stream_name):
+    sub_stream_name = stream_name
+    stream_name = PARENT_STREAM_MAP[stream_name]
+    stream_metadata = metadata.to_map(Context.get_catalog_entry(stream_name)['metadata'])
+
+    replication_key = metadata.get(stream_metadata, (), 'valid-replication-keys')[0]
+
+    bookmark_value = get_bookmark_for_stream(sub_stream_name, replication_key)
```

Comment:
What we are returning in case of the first-time sync? will the child have their own replication key?

Reply:
- We would return the start-date in case of first-time syncs. 
- Yes the child stream would have its own replication key